### ISSUE TITLE
[BUG] Docs: Incorrectly formatted Note in multiple Best Practises pages

### ISSUE THREAD

**Issue by ShubhamPalriwala** [formbricks/formbricks#2592]
### Issue Summary

On each of the Best Practises sub-pages of our documentation here: 
![image](https://github.com/formbricks/formbricks/assets/55556994/7f766b4b-9e48-43d8-91aa-78e8eab9e1df)
https://formbricks.com/docs/best-practices/interview-prompt

The notes are incorrectly formatted as seen below:

1. ![image](https://github.com/formbricks/formbricks/assets/55556994/15951531-1a2c-42ec-9896-ce526aa41ef3)

2. ![image](https://github.com/formbricks/formbricks/assets/55556994/b0b2c089-508a-49f5-b648-9ce36ca62e6f)

3. ![image](https://github.com/formbricks/formbricks/assets/55556994/0b65e36d-18aa-4ecd-9ee3-0477bcf5fc8e)

### Steps to Reproduce

Visit any page under Formbricks Best Practises and take a look at the notes

### 1st note should read like:

Formbricks Widget running? 

We assume that you have already installed the Formbricks Widget in your web-app. It’s required to display messages and surveys in your app. If not, please follow the Quick Start Guide (15mins).

### 2nd note:

Filter by attribute coming soon

We're working on pre-segmenting users by attributes. We will update this manual in the next few days.

### 3rd note:
Formbricks Widget running?

You need to have the Formbricks Widget installed to display the Feedback Box in your app. Please follow this tutorial (Step 4 onwards) to install the widget.

### Other information

Basically go through all Best Practises notes and fix them such that the heading of the note is concise & the rest of the information is in the body.

## This issue is recommended for First time contributors

We recommend this issue to be taken up by a new contributor who hasn't contribute to Formbricks yet! If you already have, keep an eye out on the other issues ;) through [oss.gg](https://oss.gg/)

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS: Arch Linux
- Node: v20.1.0
- npm: v9.6.4
```

**Comment by beonma** [comment]
Hey, im new to this repo, and would like to tackle on this issue. I’ve seen in the requirements and i don’t know if its gonna cause a problem, since im working on debian linux based system. Rephrase this

**Comment by beonma** [comment]
Hey, I'm new to this repository and interested in working on this issue. I noticed the requirements specify needing an Arch Linux-based system, but I'm currently using Debian-based Linux. I'm unsure if this will be an issue.
